src/middlewared/middlewared/plugins/service_/services/ssh.py

```diff
@@ -7,7 +7,7 @@ class SSHService(SimpleService):
 
     etc = ["ssh"]
 
-    freebsd_rc = "sshd"
+    freebsd_rc = "openssh"
     freebsd_pidfile = "/var/run/sshd.pid"
 
```

Comment:
There should also be `freebsd_procname = "sshd"` then

Reply:
Why? Shouldnt the pidfile take precedence?